core/src/main/resources/hudson/PluginManager/_installed.js

```diff
@@ -0,0 +1,49 @@
+(function () {
+  const restartRequired = document
+    .querySelector("#is-restart-required-for-completion")
+    .getAttribute("data-is-restart-required");
+
+  document.addEventListener("DOMContentLoaded", function () {
+    document
+      .querySelectorAll(".plugin-manager-toggle-switch")
+      .forEach(function (toggle) {
+        toggle.addEventListener("click", flip);
+      });
+  });
+
+  function flip(o) {
+    let btn = Event.element(o);
+
+    // trigger
+    fetch(
+      btn.getAttribute("url") +
+        "/make" +
+        (btn.checked ? "Enabled" : "Disabled"),
+      {
+        method: "post",
+        headers: crumb.wrap({}),
+      }
+    ).then((rsp) => {
+      if (!rsp.ok) {
+        $("needRestart").innerHTML = rsp.responseText;
```

Comment:
`responseText` does not exist on Response object as per https://developer.mozilla.org/en-US/docs/Web/API/Response#instance_properties. I'm unsure what was the intent here.

Reply:
my mistake, I pushed to your PR in https://github.com/jenkinsci/jenkins/pull/7912/commits/9aa54268f32dd45e0363655bef5a71bdeeb9bad6